test/model/binding.dart

```diff
@@ -521,9 +528,24 @@ class FakeAndroidNotificationHostApi implements AndroidNotificationHostApi {
       groupKey: groupKey,
       inboxStyle: inboxStyle,
       isGroupSummary: isGroupSummary,
+      messagingStyle: messagingStyle,
+      number: number,
       smallIconResourceName: smallIconResourceName,
     ));
+
+    if (tag != null && messagingStyle != null) {
+      _activeNotificationsMessagingStyle[tag] = MessagingStyle(
```

Comment:
If `messagingStyle` is null, then this should probably still update the map, right? The current notification with this tag will become a notification that has no messaging style.